airmozilla/main/templates/main/event.html

```diff
@@ -82,8 +82,8 @@
             </span>
             {% endif %}
           </p>
-
-
+    
+          
```

Comment:
Try to configure your editor so that it automatically right-strips all preceding whitespace.